### PR TITLE
Create features-list.mdx

### PR DESCRIPTION
**Pull request by ccarfi**

### CODE REVIEW

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+                "Keys", and more.
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+  </div>
```
Comment (ccarfi) [inline docs/getting-started/what-is-unlock/features-list.mdx:195]:
```suggestion
</div>
```

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                "Keys", and more.
+              </dd>
+            </dl>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+</div>
```
Comment (ccarfi) [inline docs/getting-started/what-is-unlock/features-list.mdx:195]:
```suggestion
```

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
```
Comment (ccarfi) [inline docs/getting-started/what-is-unlock/features-list.mdx:195]:
```suggestion
```

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 4]
```diff
@@ -0,0 +1,838 @@
...
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Non-expiring memberships</div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Set membership price</div>
```
Comment (julien51) [inline docs/getting-started/what-is-unlock/features-list.mdx:69]:
```suggestion
          <div className="card__header">Set membership price & currency (ERC20)</div>
```

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 5]
```diff
@@ -0,0 +1,838 @@
...
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Set platform fees</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Hooks and plugins</h2>
```
Comment (julien51) [inline docs/getting-started/what-is-unlock/features-list.mdx:577]:
```suggestion
    <h2>Onchain Hooks</h2>
```

File: docs/getting-started/what-is-unlock/features-list.mdx [hunk 6]
```diff
@@ -0,0 +1,838 @@
...
+      <div class="col">
+        <div className="card">
+          <div className="card__header">Key grant hook</div>
+        </div>
+      </div>
+    </div>
+  </div>
+  
+  <div class="full-width-section">
+    <h2>Network support</h2>
```
Comment (julien51) [inline docs/getting-started/what-is-unlock/features-list.mdx:622]: Let's remove this section and link to `/core-protocol/unlock/networks`

**Review by julien51 — APPROVED** [review]
Small nits and I would remove the networks section because I am confident this will be outdated quickly :) (we autogen the list of supported networks!)

File: docs/getting-started/what-is-unlock/features-list.mdx (hunk 6 above)
Comment (ccarfi) [reply docs/getting-started/what-is-unlock/features-list.mdx]: cool. removed.

**Pull request merged**